README.md

```diff
@@ -90,4 +90,37 @@ spec:
       - name: multiclusterhub-operator-pull-secret
 ```
 
+
+Metrics
+==================
+
+Search-v2-api also monitors and exports various metrics to Prometheus. Below is what search api currently monitors.
+
+Histograms:
+
+* `search_http_duration_seconds` - Latency of of HTTP requests in seconds.
+* `search_dbquery_duration_seconds` - Latency of DB requests in seconds.
+   * `resolveItemsFunc`- time it takes to resolve search query
+   * `resolveAutoComplete` - time it takes to resolve search complete
+   * `resolveCountFunc` - time it takes to resolve count
+* `search_dbquery_build_duration_seconds` - Latency of DB query build in seconds.
+   * `buildRbacClause` - time takes to build RBAC clause
+   * `buildRelationshipQuery` - time is takes to build relationship query
+
+
+Counters:
+
+* `search_http_total` - Total number HTTP requests.
+* `search_authn_failed_total` - The total number of authentication requests that has failed
+* `search_authz_failed_total` - The total number of authorization requests that has failed
+* `search_db_connection_failed_total` - The total number of DB connection that has failed
+* `search_db_connection_success_total` - The total number of DB connection that has succeeded
```

Comment:
Let's remove the `search_db_connection_success_total`.  We can infer this using the total requests and db connection failures.

Reply:
Done.